fireapp/lib/main.dart

```diff
@@ -29,12 +34,9 @@ import 'package:flutter_localizations/flutter_localizations.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 // Main Function
-Future<void> main() async {
-  await dotenv.load(fileName: ".env");
+void main() {
   WidgetsFlutterBinding.ensureInitialized();
   configureDependencies();
-  await Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
```

Comment:
I'm assuming these two lines should still be here? Please add back. Keep the dotenv stuff out and remove its dependency though.